packages/core/src/Config.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Service configuration model
+ */
+export interface IServiceConfig {
```

Comment:
Not sure about the `I` prefix for interfaces. Microsoft seems to discourage this: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines (under `Names`)